tyr/servers/mongo/node.py

```diff
@@ -41,13 +49,6 @@ def configure(self):
         # groups for MongoDB nodes until the security_groups argument
         # is removed.
 
-        self.security_groups = [
-            'management',
-            'chef-nodes',
-            self.envcl,
-            '{env}-mongo-management'.format(env = self.environment[0])
-        ]
-
         self.resolve_security_groups()
```

Comment:
If you set the security groups before calling `configure()` then they will automatically be resolved - no need to call it again.